### PR TITLE
added home link and fossasia link for logo

### LINKED ISSUE #460: Make the "FOSSASIA" logo link to fossasia.org
Please link the FOSSASIA logo to fossasia.org and add a "Home" link in the menu. The Home link should link to the GCI start page.

### PR DESCRIPTION
**Pull request by midhmanohar**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [ ] Squashed the commits.
 

- Fixes #460 

## Description / Changes
linked the FOSSASIA logo to fossasia.org and added a "Home" link in the menu. The Home link  linked to the GCI start page

### Before:
homebefore

### After:
homeafter

- - - - - - - - - - - -

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -69,7 +69,7 @@
     <nav class="light">
       <div class="nav-bar">
         <div class="module left">
-          <a href="./">
+          <a href="https://fossasia.org/">
```
[the remark this answers is not in the document]
Comment (midhmanohar) [reply _includes/header.html]: then whats #460 says

[the remark this answers is not in the document]
Comment (jogendra) [reply _includes/header.html]: Ohh, strange, I was not aware.

**Comment by jogendra** [comment]
Provide preview link and squash the commits

File: _includes/header.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (midhmanohar) [reply _includes/header.html]: In my machine it works without any problem
homelocalhost

but after in preview link navbar not aligned properly [ preview link ](https://midhmanohar.github.io/gci18.fossasia.org/)
homesite

can you please help me

[the remark this answers is not in the document]
Comment (jogendra) [reply _includes/header.html]: It looks fine in my machine! Try hard reloading.

[the remark this answers is not in the document]
Comment (midhmanohar) [reply _includes/header.html]: confused.

**Pull request merged**